impeller/aiks/aiks_unittests.cc

```diff
@@ -123,16 +124,47 @@ TEST_P(AiksTest, CanRenderImage) {
   ASSERT_TRUE(OpenPlaygroundHere(canvas.EndRecordingAsPicture()));
 }
 
-TEST_P(AiksTest, CanRenderInvertedImage) {
+constexpr ColorMatrix kColorInversion = {.array = {
```

Comment:
This is now defined in at least three places - it'd be nice if it could live in some common header somewhere.

Reply:
We could pull this logic all the way up too painting.dart but then we wont be able to test it with impeller yet. For the purposes of this test though it doesn't really matter what the filter is, just that it composes.